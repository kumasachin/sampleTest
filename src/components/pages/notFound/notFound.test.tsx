import React from 'react';
import NotFound from './notFound';
import Enzyme, { shallow, render, mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() })

it('renders correctly enzyme', () => {
    const wrapper = shallow(<NotFound />)

    expect(toJson(wrapper)).toMatchSnapshot();
});