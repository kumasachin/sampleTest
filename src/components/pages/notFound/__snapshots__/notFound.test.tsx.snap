// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly enzyme 1`] = `
<Fragment>
  <WithStyles(ForwardRef(Container))>
    Page not found
  </WithStyles(ForwardRef(Container))>
</Fragment>
`;
